composer.json

```diff
@@ -14,7 +14,8 @@
         "swarrot/swarrot": "^2.1.1",
         "psr/log": "~1.0",
         "symfony/framework-bundle": "~2.4|~3.0",
-        "symfony/console": "~2.4|~3.0"
+        "symfony/console": "~2.4|~3.0",
+        "symfony/yaml": ">2.0.4"
```

Comment:
Any version before Symfony 2.1 should not be allowed in constraints, as their composer.json were crappy (due to early days of composer).